src/components/Communication/CommunicationList.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { List, Datagrid, TextField, BooleanField, DateField, EditButton } from 'react-admin';
+import * as communicationConsts from './consts';
```

Comment:
What about importing each one? In my opinion, it looks better, but it's just a matter of taste. Like this:
```suggestion
import { ID, TITLE, CREATED_AT etc.. } from './consts';
``` 
and then bellow you dont need the `comunicationConsts.`